flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -160,7 +163,7 @@ def _test_create_container(self, cluster, volume_name=None):
         node = cluster.nodes[0]
         client = get_docker_client(cluster, node.public_address)
         http_port = 8080
-        host_port = find_free_port()[1]
+        host_port = 8080
```

Comment:
I fixed this properly in another branch.  Could you drop these changes so I don't have a conflict to resolve?